Marlin/ultralcd.cpp

```diff
@@ -684,11 +684,13 @@ void kill_screen(const char* lcd_msg) {
 
   #if ENABLED(MENU_ITEM_CASE_LIGHT)
 
-    extern bool case_light_on;
+    extern uint8_t case_light_brightness;  
+    bool case_light_on = (case_light_brightness >= 128) ? true : false;
     extern void update_case_light();
 
     void toggle_case_light() {
       case_light_on = !case_light_on;
+      case_light_brightness -= 255;
```

Comment:
Perhaps we should leave `case_light_brightness` as it is here. If the user "toggles" the light, it would be nice if it came back on with the same brightness level it had when it was turned off.